AgentDiscoveries-Frontend/app/src/components/admin/location-form.jsx

```diff
@@ -72,7 +90,23 @@ export default class LocationForm extends React.Component {
     }
 
     onSiteChange(event) {
-        this.setState({ siteName: event.target.value });
+        if(event.target.value.length < 20){
```

Comment:
If we use the [maxLength](https://html.com/attributes/input-maxlength/) attribute instead, I think that will give us some slightly nicer feedback for the user than the box just refusing to add characters